### PR TITLE
RM-94

### PR DESCRIPTION
**Pull request by Alinadimmokdad**
change the structure from stack form component , to data grid 
in users securitygroups tab 
and in security-groups users tab

### CODE REVIEW

File: src/pages/security-groups/Tabs/SGUsersTab.js [hunk 1]
```diff
@@ -1,179 +1,131 @@
...
-          rowId={['userId']}
-          onDelete={del}
-          isLoading={false}
-          maxAccess={maxAccess}
-          pagination={false}
-          height={300}
-        />
-      </Grow>
-    </VertLayout>
+    <>
```
Comment (imtithalhalalo) [inline src/pages/security-groups/Tabs/SGUsersTab.js:78]: no need for the empty fragment

File: src/pages/security-groups/Tabs/SGUsersTab.js [hunk 2]
```diff
@@ -1,179 +1,131 @@
...
+        infoVisible={false}
+        editMode={!!recordId}
+      >
+        <VertLayout>
+          <Grow>
+            <DataGrid
+              onChange={value => formik.setFieldValue('groups', value)}
+              value={formik.values.groups}
+              error={formik.errors.groups}
+              columns={[
```
Comment (imtithalhalalo) [inline src/pages/security-groups/Tabs/SGUsersTab.js:92]: move columns to constant

File: src/pages/security-groups/Tabs/SGUsersTab.js [hunk 3]
```diff
@@ -1,179 +1,131 @@
...
-    enabled: Boolean(recordId),
-    endpointId: AccessControlRepository.SecurityGroupUser.qry,
-    datasetId: ResourceIds.SecurityGroup
-  })
-
-  async function fetchGridData() {
-    if (!recordId) {
-      return { list: [] }
+  const postGroups = async obj => {
+    const groups = obj?.groups?.length
```
Comment (imtithalhalalo) [inline src/pages/security-groups/Tabs/SGUsersTab.js:38]: this should be > 0?

Comment (Alinadimmokdad) [reply src/pages/security-groups/Tabs/SGUsersTab.js:38]: no

**Review by imtithalhalalo — APPROVED** [review]

**Pull request merged**